COMMANDS.md

```diff
@@ -94,9 +94,10 @@ Project requirements:
 - Ensure your IAM User has the `AWSS3FullAccess` security policy enabled
 
 ##### Commands
-* `and-cli deploy aws-s3 --webpack --destination my-bucket/folder/name --profile andculture-working`
+* `and-cli deploy aws-s3 --webpack --destination my-bucket/folder/name --profile {YOUR_PROFILE_NAME}`
     * Copies webpack build artifacts from `frontend/build` and deploys them to `--destination`
     * Optionally pass `--publish` flag to perform webpack build as a part of the call
+    * `--profile` argument gives eb-cli permissions to access the s3 bucket. Defined in the initial setup (`eb-init`)
```

Comment:
Want to change wording to not use the `eb-cli` related wording. While `eb-init` _could_ be used, they are technically different. The aws s3 is run through the primary awscli.